integrations/mongodb_atlas/src/haystack_integrations/document_stores/mongodb_atlas/document_store.py

```diff
@@ -86,6 +87,10 @@ def __init__(
             MongoDBAtlasDocumentStore. For more details refer to MongoDB Atlas
             [documentation](https://www.mongodb.com/docs/atlas/atlas-search/create-index/).
         :param embedding_field: The name of the field containing document embeddings. Default is "embedding".
+        :param content_field: The name of the field containing the document content. Default is "content".
+            This field is allows defining which field to load into the Haystack Document object as content.
+            It can be particularly when integrating with an existing database for retrieval. We discourage
+            using this parameter when working with indexes created by Haystack.
```

Comment:
```suggestion
            It can be particularly useful when integrating with an existing collection for retrieval. We discourage
            using this parameter when working with collections created by Haystack.
```